test/input.txt

```diff
@@ -148,10 +148,24 @@
   clear
   list
 
+##########################################################
+# test sort command
+##########################################################
+
+  # add some names
+  add Cohn Doe p/98765432 e/cohnd@gmail.com a/Cohn street, block 123, #01-01
```

Comment:
Good job in adding a test case for the IO, maybe you can start trying out the JUnit test suite to familiarise yourself with it.